src/Microsoft.DotNet.ProjectModel/project.json

```diff
@@ -1,11 +1,11 @@
 {
-  "version": "1.0.0-*",
+  "version": "1.0.0-rc2-*",
```

Comment:
I don't understand why some of our assemblies are rc2 and some are preview1.  Is the reasoning written down somewhere?